### DISCUSSION TITLE
Permission management

### DISCUSSION THREAD

**Discussion by eglyn** [requarks/wiki#7368] [Help / Questions] [answered]
hi all :)

i am a bit confused with the permission management :/

I have many folder on root, and each folder must have its own access rules.

So, example: 
I have a folder r&d, so I create a group r&d and put inside r&d users.
In permission tab, I put read / write on pages.

And in page rules I added read / write permissions on page on url "/r&d"

Everything would be perfect, if, by default, deny for all was applied on other folder...

But I have to create forbidden rules for EVERY folder in root, for EVERY rule (12 checkbox, without any "select all"), so, I think I missed something, it can't be like that :D 

So, what I missed ? :p 

thx :)

**Comment by fnafox** [comment]
_"A more precise path will always override a less defined path."_
So deny `/` then allow `/r&d` then the group should be able to interact with `/r&d` but not the rest.
You can send each group to their own path when they login, but if you prefer having a landing on root, then you can allow that specifically, e.g. `/home`.

**Reply by eglyn** [reply]
I tried this, but it does not work: 
I denied "/" and autorise "/Communication", but users cannot connect to wiki, because home page is a part of "/"...

**Reply by fnafox** [reply]
If wiki home isn't critical to you, you can set up "Redirect on login" in each group's settings and send them to their respective home pages. Otherwise you have to allow /home as well, yeah.

**Comment by eglyn** [comment, marked as answer]
ok, I manage to do what I need with: 
![Screenshot_20241001_095803](https://github.com/user-attachments/assets/a83d6723-7d36-456e-a621-7cac76e2beb3)

That's not very intuitive, and it needs too much entries , but it works :)
